src/browser/components/MainPage.jsx

```diff
@@ -604,12 +613,11 @@ export default class MainPage extends React.Component {
   }
 
   render() {
-    const self = this;
     const tabsRow = (
       <TabBar
         id='tabBar'
         isDarkMode={this.state.isDarkMode}
-        teams={this.props.teams}
+        teams={this.state.showSettingsModal ? [] : this.props.teams}
```

Comment:
Added to maintain desired UI feature from https://github.com/mattermost/desktop/pull/1174